features/step_definitions/glance/image_create_steps.rb

```diff
@@ -0,0 +1,70 @@
+Given(/^KVM source image exists$/) do
+
+  with_scenario_config {|c|
+    @image_name         = c["image_name"]
+    @image_source       = c["image_source"]
+    @properties         = c["properties"]
+  }
+  control_node.remote_file_exists @image_source
+end
+
+Given(/^XEN HVM source image exists$/) do
+
+  with_scenario_config {|c|
+    @image_name         = c["image_name"]
+    @image_source       = c["image_source"]
+    @properties         = c["properties"]
+  }
+  control_node.remote_file_exists @image_source
+
+end
+
+Given(/^XEN PV source image exists$/) do
+
+  with_scenario_config {|c|
+    @image_name         = c["image_name"]
+    @image_source       = c["image_source"]
+    @properties         = c["properties"]
+  }
+  control_node.remote_file_exists @image_source
+
+end
+
+Given(/^glance image does not exist$/) do
+  images = control_node.openstack.image.list.map { |i| i.name}
+  expect(images).not_to include @image_name
+end
+
+When(/^I create new glance image based on jeos1$/) do
+  control_node.openstack.image.create @image_name, :properties => @properties, :disk_format => "qcow2", :public => true, :container_format => "bare", :copy_from => @image_source
```

Comment:
Please make this line shorter to fit into github viewport, 84 characters should be ok.